build/Targets/VSL.Imports.targets

```diff
@@ -69,8 +69,8 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == 'DNXCore'">
-      <BaseNuGetRuntimeIdentifier Condition="'$(OS)' == 'Windows_NT'">win</BaseNuGetRuntimeIdentifier>
-      <BaseNuGetRuntimeIdentifier Condition="'$(OS)' == 'Unix'">linux</BaseNuGetRuntimeIdentifier>
+      <BaseNuGetRuntimeIdentifier Condition="'$(OS)' == 'Windows_NT'">win7</BaseNuGetRuntimeIdentifier>
```

Comment:
Some of the NuGet packages provided by CoreFX have more specific runtimes than just "win", i.e. they only work on Win 7+. If we specify a runtime dependency of "win" then these packages won't be pulled in since they don't support arbitrary Windows runtimes, like Vista.